src/rtu/ConnectionManager.ts

```diff
@@ -58,8 +58,13 @@ export function subscribeObjectUpdates(kind, id, handler) {
   } catch (e) {
     console.log('Failed to subscribe!', request, e);
   }
-  let topic = (kind + '-update-' + id).toLowerCase().replace(/_/g, '-');
+
+  let topic =
+    kind === 'REPOSITORY_BUILD_CREATION'
+      ? (kind + '-' + id).toLowerCase().replace(/_/g, '-')
+      : (kind + '-update-' + id).toLowerCase().replace(/_/g, '-');
```

Comment:
Should we just change it on the server side to `repository-build-creation-update-$id`?

Reply:
Sounds good to me.

Reply:
See https://github.com/cirruslabs/cirrus-ci-web/pull/360/commits/7eb7e27341852bd0f5b74b958797e1068d7583ab.